shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/test/java/org/apache/shardingsphere/encrypt/metadata/EncryptTableMetaDataBuilderTest.java

```diff
@@ -175,19 +168,6 @@ private ResultSet createDataTypeResultSet() throws SQLException {
         return dataTypeResultSet;
     }
     
-    @Test
-    public void assertLoadByExistedTable() throws SQLException {
-        EncryptRule encryptRule = createEncryptRule();
-        Collection<ShardingSphereRule> rules = Arrays.asList(createSingleTableRule(), encryptRule);
-        EncryptTableMetaDataBuilder loader = (EncryptTableMetaDataBuilder) OrderedSPIRegistry.getRegisteredServices(RuleBasedTableMetaDataBuilder.class, rules).get(encryptRule);
-        when(databaseType.formatTableNamePattern(TABLE_NAME)).thenReturn(TABLE_NAME);
-        Optional<TableMetaData> actual = loader.load(TABLE_NAME, databaseType, Collections.singletonMap("logic_db", dataSource), new DataNodes(rules), encryptRule, props);
-        assertTrue(actual.isPresent());
-        assertThat(actual.get().getColumnMetaData(0).getName(), is("id"));
-        assertThat(actual.get().getColumnMetaData(1).getName(), is("pwd_cipher"));
-        assertThat(actual.get().getColumnMetaData(2).getName(), is("pwd_plain"));
-    }
-    
     @Test
     public void assertLoadByExistedTables() throws SQLException {
```

Comment:
@tuichenchuxin Please modify this method name, keep it consistent with the name of the method to be tested.